curation_concerns-models/app/services/curation_concerns/characterization_service.rb

```diff
@@ -2,9 +2,6 @@ module CurationConcerns
   # Run FITS to gather technical metadata about the content and the full text.
   # Store this extracted metadata in the characterization datastream.
   class CharacterizationService
-    include Hydra::Derivatives::ExtractMetadata
```

Comment:
That provided `filename_for_characterization` which is no longer used. `mime_type` and `uri` were required by `Hydra::Derivatives::ExtractMetadata`